presto-main/src/test/java/com/facebook/presto/sql/planner/TestLocalExecutionPlanner.java

```diff
@@ -233,162 +219,6 @@ private LocalExecutionPlan getLocalExecutionPlan(Session session)
                 new TableWriteInfo(Optional.empty(), Optional.empty(), Optional.empty()));
     }
 
-    private static class TestingOutputBuffer
```

Comment:
Is this change relevant to this PR?

Reply:
I have moved it to public scope, so that I can re-use it in a test.